configure.ac

```diff
@@ -14,7 +14,8 @@ AM_CONFIG_HEADER(config.h)
 AC_CONFIG_MACRO_DIR([m4])
 AM_MAINTAINER_MODE
 AC_SUBST(SHARED_VERSION_INFO)
-AM_ENABLE_SHARED(no) dnl shared libs cause too many headaches to be default
+# HR 20170511 need shared libraries for python interfaces
```

Comment:
this is what version control is for — no need to comment on the author of each line.